tests/tpot_tests.py

```diff
@@ -54,7 +54,7 @@
 from shutil import rmtree
 
 from sklearn.datasets import load_digits, load_boston, make_classification
-from sklearn.model_selection import train_test_split, cross_val_score, GroupKFold
+from sklearn import model_selection
```

Comment:
Since I was calling a couple more classes in `model_selection`, I just changed the import to import the entire module.

Reply:
OK, looks good.